src/network/core.rs

```diff
@@ -379,6 +388,8 @@ impl Core {
         let mut response_net_addresses: Vec<NetAddress> = Vec::new();
         // interfaces map, but we only ever expect one, for response
         let mut interfaces: HashMap<String, types::NetInterface> = HashMap::new();
+        // nameservers which can be configured for this container
+        let mut nameservers: Vec<IpAddr> = Vec::new();
```

Comment:
We do not support dns for macvlan. The gateway address will in most cases not be a host address so we cannot use that.